package.json

```diff
@@ -9,8 +9,7 @@
   "files": [
     "dist/**/*",
     "src/*",
-    "!*.spec.*",
-    "LICENSE"
```

Comment:
The `LICENSE` file is included by default, so there is no need to specify it explicitly.